scripts/ci/test_reborn_pr_test_plan.py

```
@@ -1025,6 +1025,98 @@ def test_shared_sccache_action_widens_to_exhaustive_plan(self) -> None:
+        # Full field-by-field equality, not a handful of spot checks: an
+        # exhaustive plan has ten fields besides `reasons` that a partial
+        # assertion would leave silently unpinned (e.g. `run_sandbox_docker`
+        # regressing to False on a "full" plan would pass every check this
+        # test used to make).
+        self.assertEqual(
+            plan,
+            {
+                "mode": "full",
+                "reasons": [
+                    "shared reborn action changed; this PR runs the "
+                    "exhaustive plan"
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)